modules/main/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "react-map-gl",
+  "description": "React components for MapLibre GL JS and Mapbox GL JS",
+  "version": "8.0.0-alpha.0",
+  "keywords": [
+    "mapbox",
+    "maplibre",
+    "mapbox-gl",
+    "maplibre-gl",
+    "react"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/visgl/react-map-gl.git"
+  },
+  "license": "MIT",
+  "type": "module",
+  "exports": {
+    "./mapbox": {
+      "types": "./dist/mapbox.d.ts",
+      "require": "./dist/mapbox.cjs",
+      "import": "./dist/mapbox.js"
+    },
+    "./maplibre": {
+      "types": "./dist/maplibre.d.ts",
+      "require": "./dist/maplibre.cjs",
+      "import": "./dist/maplibre.js"
+    },
+    "./mapbox-legacy": {
```

Comment:
What's the plan for legacy? Is it just the previous codebase until we've separated out Mapbox and Maplibre completely?

Reply:
We need it to support mapbox-gl v1.